### PR TITLE
Created page for BTL ep 10

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Content and images + embedding youtube video

### CODE REVIEW

**Review by mansona — APPROVED** [review]
looks good

File: _videos/2021-08-04-built-to-last-daniel-beutler.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+moreText: more Expert insights
+image: /assets/images/resources/videos/2021-08-04-built-to-last-10-daniel-beutler/daniel.jpeg
+duration: '25:40'
+teaser:
+  Daniel talks about accepting the necessity for change in a growing company to
+  ensure the business keeps scaling.
+
+description:
+  simplabs' founder Marco Otte-Witte sat down for a conversation with
+  interviewed Daniel Beutler, an executive coach and advisor and former
```
Comment (marcoow) [inline _videos/2021-08-04-built-to-last-daniel-beutler.md]:
```suggestion
  Daniel Beutler, an executive coach and advisor and former
```

File: _videos/2021-08-04-built-to-last-daniel-beutler.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  President of Trainline International.
+
+og:
+  image: /assets/images/resources/videos/2021-08-04-built-to-last-10-daniel-beutler/og-image.png
+---
+
+simplabs' founder Marco Otte-Witte, interviewed Daniel Beutler, an executive
+coach and advisor, that helps companies and startups to scale and grow. Before
```
Comment (marcoow) [inline _videos/2021-08-04-built-to-last-daniel-beutler.md]:
```suggestion
coach and advisor, who helps companies and startups to scale and grow. Before
```

File: _videos/2021-08-04-built-to-last-daniel-beutler.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+that, he was President of Trainline International, which he led through becoming
```
Comment (marcoow) [inline _videos/2021-08-04-built-to-last-daniel-beutler.md]:
```suggestion
that, Daniel was President of Trainline International, which he led through becoming
```

**Comment by marcoow** [comment]
@emmasofiah2o I reduced the file size of the images – the photos never need to be larger than 1080x1080 or so

**Review by marcoow — APPROVED** [review]

**Pull request merged**